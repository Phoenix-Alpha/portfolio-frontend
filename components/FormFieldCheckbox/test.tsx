import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { FormFieldCheckbox } from '.'

describe('<FormFieldCheckbox />', () => {
  const name = 'test-name'

  it('should renders correct <FormFieldCheckbox />', () => {
    const wrapper = shallow(<FormFieldCheckbox name={name} />)

    expect(wrapper.find('Field').prop('name')).toMatch(name)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
