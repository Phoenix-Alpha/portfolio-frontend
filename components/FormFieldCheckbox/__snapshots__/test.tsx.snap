// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormFieldCheckbox /> should renders correct <FormFieldCheckbox /> 1`] = `
<Field
  name="test-name"
>
  <Component />
</Field>
`;
